### ISSUE TITLE
Basic example in README.md not working

### ISSUE THREAD

**Issue by dinko-pehar** [phenopolis/pluto#58]
Hello.

Thank you for making this kind of library open source. I'm trying to test it to see how it works. Can you provide more example on how to use this library ? For example, Crystal lang Redis library has lot of examples that really represent in good way how to use it. Maybe you can provide set of those as well.
I'm trying to run example in` README.md` , but I get:

> Error: undefined constant `Pluto::ImageRGBA`

Updating basic example would be enough for me and others to get started quickly without debugging.

Thank you 🚀 .

**Comment by sanks64** [comment]
Hello. This is because Shards uses the latest release (0.2.2 at the time) by default and the example is for the `main` branch.

Do one of these:

- Switch to `main` in `shard.yml`:

  ```yaml
  dependencies:
    pluto:
      github: phenopolis/pluto
      branch: main
  ```

- Change `ImageRGBA` to `RGBAImage`, open [the 0.2.2 tag](https://github.com/phenopolis/pluto/tree/v0.2.2) for docs.

I agree that it would be a good idea to provide `examples/` in the long-term.

**Comment by sanks64** [comment]
I just released 0.3.0, so simply updating should fix the issue.

**Comment by dinko-pehar** [comment]
Thank you @sanks64 . I tried yesterday, I had some errors again. I will check again soon and update this comment if it works.
